somatic_llr_filter.py

```diff
@@ -133,6 +146,17 @@ def create_vcf_writer(args, vcf_reader):
         od = OrderedDict([('ID', args.somatic_field), ('Number', '0'), ('Type', 'Flag'), ('Description', 'Is a somatic mutation')])
         new_header.add_info_line(od)
 
+    #check/add FILTER field in header
+    if not args.llr_threshold == None: #filtering may not even be specified
+        if args.filter_field in vcf_reader.header.filter_ids():
+            if not args.overwrite:
+                vcf_reader.close()
+                raise Exception("FILTER {} already exists. Choose a different --filter-field, or use the --overwrite flag to retain this filter description and overwrite values".format(args.filter_field))
+        else:
```

Comment:
What should this do when the field is found and `args.overwrite` is true?

Reply:
nothing!  The existing filter field definition will be kept

Reply:
Right! Makes sense...I just confused myself.